dsgrid/config/date_time_dimension_config.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from datetime import datetime
+
+import pandas as pd
+import pyspark.sql.functions as F
+
+from dsgrid.dimension.time import TimeZone
+from dsgrid.dimension.time import TimeZone, make_time_range
+from dsgrid.exceptions import DSGInvalidDataset
+from .dimensions import DateTimeDimensionModel
+from .time_dimension_base_config import TimeDimensionBaseConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+class DateTimeDimensionConfig(TimeDimensionBaseConfig):
+    """Provides an interface to a DateTimeDimensionModel."""
```

Comment:
I understand that this name is more consistent with other time packages, but are we going to have to re-register all of the time dimensions again?

Is it time to implement more backward compatibility with respect to software versions?